src/bidiMapper/domains/script/realmStorage.ts

```diff
@@ -117,6 +136,15 @@ export class RealmStorage {
   deleteRealms(filter: RealmFilter) {
     this.findRealms(filter).map((realm) => {
       this.#realmMap.delete(realm.realmId);
```

Comment:
may it would be more logical to send event from the realm itself rather than from the storage?

Reply:
We should as it follows the same logic as the `BrowsingContextImpl`. Same method `.delete`